azext_iot/tests/iothub/core/test_iothub_storage_int.py

```diff
@@ -224,6 +224,9 @@ def test_system_identity_storage(self):
 
         self.assign_storage_role_if_needed(hub_id)
 
+        # give time to finish job
+        sleep(30)
```

Comment:
Is this sleep that beneficial since `assign_storage_role_if_needed()` already has `sleep(30)` as a last step? Wondering if role propagation is taking too long, then we can up the sleep (in the assign_storage_role_if_needed function) to 60 sec.

Reply:
there was an extra sleep here https://github.com/vilit1/azure-iot-cli-extension/blob/903f950e238b6269e8e4e06c371c35d5d61e6aac/azext_iot/tests/iothub/core/test_iothub_storage_int.py#L321 so I figured may as well play it safe